### ISSUE TITLE
should input_quant be Int8ActPerTensorFloat?

### ISSUE THREAD

**Issue by ardeal** [Xilinx/brevitas#710]
Hi, 

for the following code, the output of `self.pool1` is quantized, should the `input_quant` of `self.conv2` be `Int8ActPerTensorFloat` or `None`?

One more question is:
is there any example code about how to use `BatchNorm2dToQuantScaleBias`, `QuantReLU` and other ops?

```python

self.conv1 = qnn.QuantConv2d(in_channels, cfg[0], kernel_size=(3,3), padding=1, bias=True, input_quant=Int8ActPerTensorFloat, weight_bit_width=8, bias_quant=Int8Bias, output_quant=Int8ActPerTensorFloat)
self.bn1   = qnn.BatchNorm2dToQuantScaleBias(cfg[0], weight_quant_type=Int8ActPerTensorFloat, weight_bit_width=8)
self.relu1 = qnn.QuantReLU(quant_type=Int8ActPerTensorFloat, bit_width=8)
self.pool1 = qnn.QuantMaxPool2d(kernel_size=2, stride=2, output_quant=Int8ActPerTensorFloat)

self.conv2 = qnn.QuantConv2d(cfg[0], cfg[2], kernel_size=3, padding=1, bias=True, input_quant=Int8ActPerTensorFloat, weight_bit_width=8, bias_quant=Int8Bias, output_quant=Int8ActPerTensorFloat)

```
